pkg/dist/go_plugin_build_manifest

```diff
@@ -0,0 +1,373 @@
+ece270190e2c60c26f8d9ebc9c16ae3aad3babc0d20aa8dfca15d0d935ace0a8:Magefile.go
```

Comment:
Should we be committing dist files? Seems like something that was generated by the build